hera_mc/cm_sysdef.py

```diff
@@ -295,3 +295,18 @@ def get_ports(self, pol, part_type):
                     elif port[0].lower() == pol[0].lower():
                         port_dict[dir].append(port.upper())
         return port_dict
+
+    def node(self, node_nums):
```

Comment:
Is this also just re-arranged from somewhere else or is it new code? I don't see it anywhere else. It needs a docstring in either case.

Reply:
This is the new code for this PR.  I've added a docstring